### PR TITLE
Eliminates line breaks before/after xliff tag

### PR DESCRIPTION
**Pull request by mariotaku**
`xliff` is a placeholder tag for localization files: https://developer.android.com/distribute/tools/localization-checklist.html

This code change eliminates line breaks before/after that tag, which is a huge improvement for code style.

<- Before | After ->

image

### CODE REVIEW

**Comment by roana0229** [comment]
👀  I confirm tonight.

File: src/XmlSorterUtil.java [hunk 1]
```diff
@@ -32,14 +33,24 @@ public static Document parseStringToDocument(@NotNull String content) throws Exc
...
-        outputFormat.setIndenting(true);
-        outputFormat.setIndent(indent);
-        Writer out = new StringWriter();
-        XMLSerializer serializer = new XMLSerializer(out, outputFormat);
-        serializer.serialize(document);
-        return out.toString();
+    public static String prettyStringFromDocument(@NotNull Document document, int indent, boolean insertEncoding)
+            throws IOException {
+        try {
+            TransformerFactory tFactory = TransformerFactory.newInstance();
```
Comment (roana0229) [inline src/XmlSorterUtil.java:39]: Awesome 👍

**Review by roana0229 — APPROVED** [review]
Thank you for PR!
LGTM 😄

**Comment by mariotaku** [comment]
Thanks! Your plugin is awesome!

**Pull request merged**